src/profile/style.css

```diff
@@ -157,6 +157,19 @@ nav h1::before {
   border-right: 1px solid #c4c4c4;
 }
 
+.oldDiff {
+  background-color: #fd5757;
+  padding: 4px;
+  border-radius: 2px;
+  color: #fff;
```

Comment:
There is already a variable for this please use that

Reply:
Can you please check now?